src/phaser/dino-game.js

```diff
@@ -379,15 +379,11 @@ class DinoGame extends Phaser.Scene {
                     `doge-coin`
                 )
                 .setOrigin(0, 1);
-            obsticle.play('doge-coin-fly', 1);
+            obsticle.play('enemy-dino-fly', 1);
```

Comment:
obsticle.play('doge-coin-fly', 1); this should be there , please update @mahdiyeh-fs